### PR TITLE
dependencies and visibility improvement

### PR DESCRIPTION
**Pull request by GalaxySciTech**
- Replace dependencies to become hardhat adaptation
- library visibility becomes internal to prevent redeployment

### CODE REVIEW

File: forge/src/forge.rs [hunk 1]
```diff
@@ -31,7 +31,7 @@ static PROJECT: Lazy<Project> = Lazy::new(|| {
         root.push("lib/openzeppelin-contracts/contracts");
         paths.remappings.push(Remapping {
             context: None,
-            name: "openzeppelin/".to_string(),
+            name: "@openzeppelin/contracts/".to_string(),
```
Comment (seunlanlege) [inline forge/src/forge.rs:34]: this isn't necessary

File: remappings.txt [hunk 2]
```diff
@@ -1 +1 @@
-openzeppelin/=lib/openzeppelin-contracts/contracts/
+@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
```
Comment (seunlanlege) [inline remappings.txt:1]: prefer we keep this too

File: src/MerkleMountainRange.sol [hunk 3]
```diff
@@ -34,7 +34,7 @@ library MerkleMountainRange {
     /// @param leaves a list of mmr leaves to prove
     /// @return boolean if the calculated root matches the provides root node
     function VerifyProof(bytes32 root, bytes32[] memory proof, MmrLeaf[] memory leaves, uint256 mmrSize)
-        public
+        internal
```
Comment (seunlanlege) [inline src/MerkleMountainRange.sol:37]: For our project needs, we can't embed this contract. It needs to be deployed seperately as it is too large

Comment (GalaxySciTech) [reply src/MerkleMountainRange.sol:37]: > For our project needs, we can't embed this contract. It needs to be deployed seperately as it is too large

i think if use public  , the type library need change to type contract

File: forge/src/forge.rs (hunk 1 above)
Comment (GalaxySciTech) [reply forge/src/forge.rs:34]: i change back

File: remappings.txt (hunk 2 above)
Comment (GalaxySciTech) [reply remappings.txt:1]: i added

**Pull request closed without merging**